cmd/util/cmd/common/clusters.go

```diff
@@ -47,24 +61,26 @@ func constructClusterAssignment(partnerNodes, internalNodes []model.NodeInfo) (f
 		log.Fatal().Err(err).Msg("could not shuffle internals")
 	}
 
```

Comment:
I think a nice addition here would be the following sanity check:
* the heuristic we apply here only works correctly if all collectors are equally weighted. The bootstrapping code generates them currently. Nevertheless, adding a sanity check here wouldn't hurt to prevent accidentally producing invalid outputs


```suggestion

	// The following is a heuristic for distributing the internal collector nodes (private staking key available
	// to generate QC for cluster root block) and partner nodes (private staking unknown). We need internal nodes
	// to control strictly more than 2/3 of the cluster's total weight.
	// The following is a heuristic that distributes collectors round-robbin across the specified number of clusters.
	// This heuristic only works when all collectors have equal weight! The following sanity check enforces this:
	if !(0 < len(partnerNodes) && len(partnerNodes) < 2*len(internalNodes)) {
		return nil, nil, fmt.Errorf("requiring at least x>0 number of partner nodes and y > 2x number of internal nodes, but got x,y=%d,%d", len(partnerNodes), len(internalNodes))
	}
	// sanity check ^ enforces that there is at least one internal node, hence `internalNodes[0].InitialWeight` is always a valid reference weight
	refWeight := internalNodes[0].InitialWeight
	
```

then in [line 70](https://github.com/onflow/flow-go/blob/6bc838f5d7ca95e6cb0658e3fde7cc6e5123873b/cmd/util/cmd/common/clusters.go#L70) and [line 77](https://github.com/onflow/flow-go/blob/6bc838f5d7ca95e6cb0658e3fde7cc6e5123873b/cmd/util/cmd/common/clusters.go#L70) we would include the following sanity check:
```golang
if node.InitialWeight != refWeight {
  return nil, nil, fmt.Errorf("current implementation requires all collectors (partner & interal nodes) to have equal weight")
}
```

Would be great if you could add this. Thanks